loco/hello-world/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Loco",
+    "dockerComposeFile": "docker-compose.yml",
+    "service": "app",
+    "workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}",
+    "forwardPorts": [
+        3000
```

Comment:
cargo-shuttle runs on 8000 by default. Should that be added here?

Reply:
change